### ISSUE TITLE
Problems about content loss in second pass

### ISSUE THREAD

**Issue by NikoNairre** [owenzlz/Deep_Image_Blending#25]
The content loss in stage loss is always 0.0000 during training. I found that code in run.py
```python
blend_features_style = vgg(mean_shift(first_pass_img))
content_features = vgg(mean_shift(first_pass_img))
content_loss = content_weight * mse(blend_features_style.relu2_2, content_features.relu2_2)
```
In this way, these two features are totally identical. What's the correct code to compute content loss for $I_{BR}$ and $I_{B}$?
